src/pages/distroSettings/tabs/HostTab/getFormSchema.tsx

```diff
@@ -53,95 +63,118 @@ export const getFormSchema = ({
             workDir: {
               type: "string" as "string",
               title: "Working Directory",
-            },
-            setupScript: {
-              type: "string" as "string",
-              title: "Setup Script",
+              minLength: 1,
             },
             setupAsSudo: {
               type: "boolean" as "boolean",
               title: "Run script as sudo",
```

Comment:
When I was developing I thought it was never super clear that "Run script as sudo" is attached to the setup script when it was located elsewhere in the form. I added a max width to our Spruce Form elements instead; I think this is worth doing regardless because forms can get pretty wonky/hard to follow on ultrawide screens.